components/carousel/private/carousel-item.jsx

```diff
@@ -21,7 +21,7 @@ const CarouselItem = (props) => (
 		className="slds-carousel__panel slds-m-horizontal_xx-small slds-list_horizontal"
 		role="tabpanel"
 		aria-hidden="false"
-		aria-labelledby={`indicator-id-${props.id}`}
+		aria-labelledby={`indicator-id-${props.panelIndex}`}
```

Comment:
@kevinparkerson This is going to need to be unique to the page. I'm not sure it is now if there are two carousels on the page.